### PR TITLE
Fix docker image file modes

### PR DESCRIPTION
**Pull request by jensl**
Make files in `/usr/include/google/protobuf` world readable in the docker image.

Also update gRPC and Protobuf versions that are installed, to fix the docker image build.

### CODE REVIEW

**Comment by smaye81** [comment]
Thank for being patient on this.  If you are still interested in this PR, can you rebase dev and update?

**Comment by jensl** [comment]
PR rebased onto current dev.

File: etc/docker/testing/bin/test.sh [hunk 1]
```diff
@@ -59,15 +59,15 @@ check_dir_not_exists() {
 
 check_env GOGO_PROTOBUF_VERSION 1.2.1
 check_env GOLANG_PROTOBUF_VERSION 1.3.1
-check_env GRPC_VERSION 1.19.1
+check_env GRPC_VERSION 1.25.0
 check_env GRPC_GATEWAY_VERSION 1.8.5
 check_env GRPC_WEB_VERSION 1.0.4
-check_env PROTOBUF_VERSION 3.6.1
+check_env PROTOBUF_VERSION 3.10.1
```
Comment (smaye81) [inline etc/docker/testing/bin/test.sh]: Can you bump this to 3.11.2

File: etc/docker/testing/bin/test.sh [hunk 2]
```diff
@@ -59,15 +59,15 @@ check_dir_not_exists() {
...
 check_env TWIRP_VERSION 5.7.0
 check_env YARPC_VERSION 1.37.3
 check_env PROTOTOOL_PROTOC_BIN_PATH /usr/bin/protoc
 check_env PROTOTOOL_PROTOC_WKT_PATH /usr/include
-check_command_output "libprotoc 3.6.1" protoc --version
+check_command_output "libprotoc 3.10.1" protoc --version
```
Comment (smaye81) [inline etc/docker/testing/bin/test.sh]: This will probably also need changed

**Comment by smaye81** [comment]
Can you run `make dockerall` just to verify all works correctly (once making the version changes suggested)?

**Comment by smaye81** [comment]
FYI, if you rebase, you should get all the latest dependency updates from the `dev` branch.

**Comment by smaye81** [comment]
Ping on this.

**Pull request closed without merging**